### ISSUE TITLE
dodgy font-size declarations

### ISSUE THREAD

**Issue by ericscheid** [naturalcrit/homebrewery#1834]
Most of the `font-size` declarations in the v3 stylesheet are in `cm`, but there are a few which are not, and one which is entirely invalid (no unit specified at all). 

The `font-size: .325;` declartion (on `.page code` is likely meant to be `font-size: .325cm;`, just a tidge smaller than the regular font-size of `.34cm`. (Adjusting for optical scaling of the monospaced font?)

Should be easy to convert them all to `cm` (though arguably they might not need be).

```
  font-size: .34cm;
  font-size: smaller; /* .page sup  */
  font-size: smaller; /* .page sub */
  font-size: .89cm;
  font-size: 3.5cm;
  font-size: .75cm
  font-size: .575cm;
  font-size: .458cm
  font-size: .423cm;
  font-size: .318cm;
  font-size: .318cm;
  font-size: .375cm
  font-size: .318cm;
  font-size: .375cm
  font-size: .27cm;
  font-size: 1.3em; /* .page .artist h5 */
  font-size: 120px; /* .page .watermark */
  font-size: .318cm;
  font-size: .62cm;
  font-size: .304cm;
  font-size: .9em; /* .page .pageNumber */
  font-size: .8em; /* .page .footnote */
  font-size: .325; /* .page code ⇐ THIS IS AN ERROR AS NO UNITS ARE SPECIFIED */
  font-size: .318cm;
  font-size: .352cm;
  font-size: .34cm;
```

**Comment by ericscheid** [comment]
And the legacy `font-size` declarations:

```
  font-size: .317cm;
  font-size: smaller; /* .phb sup */
  font-size: smaller; /* .phb sub */
  font-size: .987cm;
  font-size: 10em;    /* .phb h1+p::first-letter */
  font-size: .705cm
  font-size: .529cm;
  font-size: .458cm
  font-size: .423cm;
  font-size: 10pt     /* .phb table */
  font-size: .352cm;
  font-size: .352cm
  font-size: .9em;    /* .phb .pageNumber */
  font-size: .8em;    /* .phb .footnote */
  font-size: .352cm;
  font-size: .27cm;
  font-size: 1.3em;   /* .phb .artist h5 */
```

**Comment by 5e-Cleric** [comment]
this should be fixed for v3 with #2981, recently merged, closing.
